### ISSUE TITLE
wait_for task in main.yml in kibana get timeout

### ISSUE THREAD

**Issue by afeefghannam89** [NETWAYS/ansible-collection-elasticstack#169]
Wait for kibana to start task get time out in GitHub action while using Rocky8 and Elastic 8. The pipeline will be terminated after that. I tried to increase the time out or figure out the reason, but did not success.

When using Elastic7 on Rocky8 the task does not face any problem. DNS resolve and port are ok. For more information about the look at the workflow file `test_full_stack.yml`

**Comment by afeefghannam89** [comment]
Also wait_for in other container like Debian 10 get timeout while merging in main.

**Comment by widhalmt** [comment]
Yes, that's a problem I stumbled upon in the past, too. Honestly I haven't figured out, what's wrong. But yes, we need to look into that.

**Comment by afeefghannam89** [comment]
We use now Rocky 9. This time out issue doesn't appear anymore.
